src/components/catalog-filter/catalog-filter.component.ts

```diff
@@ -0,0 +1,12 @@
+import {CatalogFilterController} from './catalog-filter.controller';
+
+export const catalogFilter: angular.IComponentOptions = {
+  bindings: {
+    config: '<',
+    filterOnKeyword: '<',
+    clearFilters: '<',
```

Comment:
You can still use `$scope`.

`clearFilters` is not a variable, it's an action. It doesn't feel right to treat it as a variable. No reason not to use `$scope.broadcast` with components.